### PR TITLE
[Overflow-185] Create Question list page

### PR DESCRIPTION
**Pull request by jescabillas**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-185

## Commands
None

## Pre-conditions
None

## Expected Output
Question list page

## Notes
None

## Screenshots
image

### CODE REVIEW

File: frontend/components/templates/layouts/index.tsx [hunk 1]
```diff
@@ -3,15 +3,18 @@ import { useRouter } from 'next/router'
 import Navbar from '../../organisms/Navbar'
 import Sidebar from '../../organisms/Sidebar'
 import ProvidersWrapper from '../ProvidersWrapper'
+import { array } from 'yup'
```
Comment (IanUrriza10) [inline frontend/components/templates/layouts/index.tsx]: Accidental import?

File: frontend/components/templates/layouts/index.tsx [hunk 2]
```diff
@@ -3,15 +3,18 @@ import { useRouter } from 'next/router'
...
 type LayoutProps = {
     children: JSX.Element
 }
 
 const Layout = ({ children }: LayoutProps) => {
     const router = useRouter()
+    const hideRightSidebarInPages = ['/questions/[id]']
 
     const routeIfLoginPathCheck = router.asPath === '/login' || router.asPath === '/login/check'
+    console.log(router.basePath)
```
Comment (IanUrriza10) [inline frontend/components/templates/layouts/index.tsx]: Remove console.log

**Review by nicoleamber — APPROVED** [review]

File: frontend/components/atoms/Button/index.tsx [hunk 3]
```diff
@@ -1,27 +1,61 @@
...
-            return `text-red-700 border-red-500 focus:ring-red-600 hover:bg-rose-200`
-        } else if (usage === 'light') {
-            return `text-gray-700 bg-white focus:ring-gray-600 hover:bg-slate-100`
-        }
+}: Props) => {
+    let htmlClasses: string = ''
+
+    switch (usage) {
+        case 'primary':
+            htmlClasses =
```
Comment (kenorenzsun) [inline frontend/components/atoms/Button/index.tsx]:
```suggestion

    switch (usage) {
        case 'primary':
            return
```

**Review by jules-lucero — APPROVED** [review]

**Comment by ipilikent** [comment]
### Description
- The description will be lowered a bit
- cut the left side line a little
- The username will be lowered slightly
- Lower the pagination of the questions

Screenshot:

Screenshot 2023-02-08 152545

Screenshot 2023-02-08 152458

Screenshot 2023-02-08 152420

Screenshot 2023-02-08 152340


### Expected Behavior
- The Description, Line, Username, Pagination must be correct in their placement

**Comment by ipilikent** [comment]
## Proof
- All displayed the same as the Figma design
- The Ask a Question button is displayed 
- The Pagination is displayed
- Votes, Answers and Views is displayed
- Question Title, Description, Tags and Username is displayed
- The Date Bar and Answered Bar is displayed


<details>

185 - Markup Question List Page

</details>

Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=48930172
-----
- Total Test time: 1 hr.
- Total Comments: 1
- Total Iterations: 1

**Pull request merged**